spec/views/granules/required_form_params_spec.rb

```diff
@@ -5,6 +5,7 @@
 
   it 'does not allow granule searches with empty UniqueID and ShortName and CollectionID' do
     VCR.use_cassette 'views/granule/allRequiredFormParamsEmpty', :record => :once, :decode_compressed_response => true do
+			puts granules_path
```

Comment:
should this line up with the line below?

Reply:
nope

Reply:
It should be gone, debug code.